pkgs/data/fonts/cozette/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv, fetchurl, mkfontscale }:
+
+let
+  version = "1.5.1";
+  releaseUrl =
+    "https://github.com/slavfox/Cozette/releases/download/v.${version}";
+in stdenv.mkDerivation rec {
+  srcs = [
+    (fetchurl {
+      url = "${releaseUrl}/cozette.otb";
+      sha256 = "05k45n7jar11gnng2awpmc7zk9jdlzd6wz87xx49cp75jm4z9xm8";
+    })
+
+    (fetchurl {
+      url = "${releaseUrl}/CozetteVector.otf";
+      sha256 = "1sqhnjpizn1wi26lc7z2zml7yr7zkcpa72mh1drvd74rlcs1ip30";
+    })
+
+    (fetchurl {
+      url = "${releaseUrl}/CozetteVector.ttf";
+      sha256 = "1q4ml8shv9lmyc6bwhffwvbvl92s73j7xkb0rkqvci4f0zbz7mcy";
+    })
+  ];
+
+  nativeBuildInputs = [ mkfontscale ];
+
+  sourceRoot = "./";
+
+  unpackCmd = ''
+    otName=$(basename $(stripHash $curSrc))
+    cp $curSrc ./$otName
+  '';
+
+  installPhase = ''
+
+    mkdir -p $out/share/fonts/opentype
+    cp -a *.otf $out/share/fonts/opentype/
+
+    mkdir -p $out/share/fonts/truetype
+    cp -a *.ttf $out/share/fonts/truetype/
+
+    mkdir -p $out/share/fonts/misc
+    cp -a *.otb $out/share/fonts/misc/
+
+    mkfontdir "$out/share/fonts/misc"
+    mkfontscale "$out/share/fonts/truetype"
+    mkfontscale "$out/share/fonts/opentype"
+  '';
+
+  outputs = [ "out" ];
```

Comment:
Not that this really matters, but if you don't specify any `outputs`, it defaults to `out`.